index.js

```diff
@@ -62,7 +62,7 @@ module.exports = class MongooseTrailpack extends DatastoreTrailpack {
         schema.statics = model.statics
         schema.methods = model.methods
 
-        model.onSchema(schema)
+        model.onSchema(schema, this.app)
```

Comment:
@LobeTia It's made like this because of mongoose.
We couldn't add `pre/post` hooks or `virtual` fields before out schema is created. 
So we have to create schema first and after this give users an ability to manipulate with it before `Model` created from it.